.github/workflows/build_and_test.yml

```diff
@@ -89,9 +89,39 @@ jobs:
           npm install -g @stellar/anchor-tests
           stellar-anchor-tests --home-domain $HOME_DOMAIN --seps 1 10 12 31 38 --sep-config platform/src/test/resources/stellar-anchor-tests-sep-config.json
 
+  end_to_end_test:
+    needs: [build_and_test]
+    runs-on: ubuntu-latest
+    name: End to End Tests
+    env:
+      PR_NUMBER: ${{github.event.pull_request.number}}
+    steps:
+      - uses: actions/checkout@v3
+
+      # Find the server endpoint home domain to run the End to End tests on.
+      - name: Find Home Domain (Preview or Dev)
+        id: endpoint-finder
+        run: |
+          export HOME_DOMAIN=anchor-sep-server-dev.stellar.org
+          if [ ! -z "$PR_NUMBER" ]
+          then
+            export HOME_DOMAIN=anchor-sep-pr$PR_NUMBER.previews.kube001.services.stellar-ops.com
+          fi
+          echo HOME_DOMAIN=$HOME_DOMAIN
+          echo "::set-output name=HOME_DOMAIN::$HOME_DOMAIN"
+
+      - name: Install Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Run End to End Tests
+        run: |
+          python -m pip install --upgrade pip
+          if [ -f end-to-end-tests/requirements.txt ]; then pip install -r end-to-end-tests/requirements.txt; fi
+          python end-to-end-tests/end_to_end_tests.py --domain ${{ steps.endpoint-finder.outputs.HOME_DOMAIN }} --tests all --secret SDLE32FFTGZT4O6MAUHUUEUNFK3MTH4Q2VJR2TQFYCECJBGDSE2LJILV
   complete:
-    needs: [build_and_test, sep_validation_suite]
+    needs: [build_and_test, sep_validation_suite, end_to_end_test]
     runs-on: ubuntu-latest
     steps:
-      - if: contains(needs.*.result, 'failure')
+      - if: contains(needs.*.result, 'failure') || failure()
```

Comment:
Is this new conditional `|| failure()` needed?

Reply:
>>>
1. having the tests create and setup a brand new account, so the account only has the balance needed.
2. Or store this secret safely – ops can help with that – and still have a step to re-create and configure the account if needed.
>>>
1 - i considered this but the account would still need to be funded with USDC and JPYC. we could have the account try to fund itself with the USDC/XLM and JPYC/XLM offers but those arent always guaranteed to be available since someone could just deplete those offers. 
2 - ops added the secrets to the repo but secrets are only accessible by the repo (when code is merged) and not on forks (PR previews)

Reply:
just tested without and it looks like the contains() statement still catches the failure. i'll remove it.